AdblockPlusSafari/AdblockPlus.h

```diff
@@ -17,22 +17,28 @@
 
 #import <Foundation/Foundation.h>
 
+extern NSString *_Nonnull AdblockPlusActivated;
+
 @interface AdblockPlus : NSObject
 
+@property (nonatomic, strong, readonly) NSUserDefaults *__nonnull adblockPlusDetails;
+
+- (NSString *__nonnull)group;
+
+- (NSString *__nonnull)contentBlockerIdentifier;
+
+- (NSString *__nonnull)backgroundSessionConfigurationIdentifier;
+
 @property (nonatomic) BOOL enabled;
 
 @property (nonatomic) BOOL acceptableAdsEnabled;
 
 @property (nonatomic) BOOL activated;
 
-@property (nonatomic) BOOL reloading;
-
-- (void)setEnabled:(BOOL)enabled reload:(BOOL)reload;
-
-- (void)setAcceptableAdsEnabled:(BOOL)enabled reload:(BOOL)reload;
+@property (nonatomic) NSInteger installedVersion;
```

Comment:
You are right. Actually, it seemed to me as good idea to keep version of filter list, because I known how many times it was updated. For the end user product it is not really useful. I introduced dirty property.